README.md

```diff
@@ -1,20 +1,21 @@
 # PyC8
 
-Welcome to the GitHub page for **pyC8**, a Python driver for the Digital Edge Fabric.
+Python SDK for the Macrometa Global Data Network.
 
-### Features
+# Compatibility
 
+- Python versions 3.4, 3.5 and 3.6 are supported.
```

Comment:
I think we support up to 3.10 without issues.

Reply:
👍  Yea, I have been testing with 3.10 as well, thanks